### PR TITLE
Renamed OutputMode.EMIT_ALL_SITES to EMIT_ALL_ACTIVE_SITES

### PR DESCRIPTION
**Pull request by michaelgatzen**
- Modified GenotypeGVCFsEngine.OutputMode enum documentation to reflect this change

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/OutputMode.java [hunk 1]
```diff
@@ -10,7 +10,8 @@
     /** produces calls at variant sites and confident reference sites */
     EMIT_ALL_CONFIDENT_SITES,
 
-    /** produces calls at any callable site regardless of confidence; this argument is intended only for point
-     * mutations (SNPs); it will not produce a comprehensive set of indels. */
-    EMIT_ALL_SITES
+    /** produces calls at any callable site regardless of confidence. This does not necessarily output calls for all
```
Comment (jamesemery) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/OutputMode.java]: `produces calls at any callable site regardless of confidence` -> `Produces calls at any region over the activity threshold regardless of confidence`

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingEngine.java [hunk 2]
```diff
@@ -376,7 +376,7 @@ protected final boolean hasTooManyAlternativeAlleles(final VariantContext vc) {
     }
 
     protected boolean emitAllSites() {
```
Comment (jamesemery) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingEngine.java]: Put a comment here explaining this in somewhat better detail I think.

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/OutputMode.java (hunk 1 above)
Comment (michaelgatzen) [reply src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/OutputMode.java]: done

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingEngine.java (hunk 2 above)
Comment (michaelgatzen) [reply src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingEngine.java]: I put a similar comment there and referred to the OutputMode documentation. Do you want more detail?
I also renamed this function to `emitAllActiveSites()`, just to be consistent.

**Pull request merged**